2022-02-09 06:28:46,404 INFO    MainThread:2844 [wandb_setup.py:_flush():71] setting env: {}
2022-02-09 06:28:46,404 INFO    MainThread:2844 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-09 06:28:46,405 INFO    MainThread:2844 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220209_062846-16zzxr2h/logs/debug.log
2022-02-09 06:28:46,405 INFO    MainThread:2844 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220209_062846-16zzxr2h/logs/debug-internal.log
2022-02-09 06:28:46,405 INFO    MainThread:2844 [wandb_init.py:init():404] calling init triggers
2022-02-09 06:28:46,406 INFO    MainThread:2844 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-02-09 06:28:46,406 INFO    MainThread:2844 [wandb_init.py:init():449] starting backend
2022-02-09 06:28:46,406 INFO    MainThread:2844 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-09 06:28:46,423 INFO    MainThread:2844 [backend.py:ensure_launched():199] starting backend process...
2022-02-09 06:28:46,430 INFO    MainThread:2844 [backend.py:ensure_launched():204] started backend process with pid: 2927
2022-02-09 06:28:46,431 INFO    MainThread:2844 [wandb_init.py:init():458] backend started and connected
2022-02-09 06:28:46,434 INFO    MainThread:2844 [wandb_init.py:init():519] updated telemetry
2022-02-09 06:28:46,436 INFO    MainThread:2844 [wandb_init.py:init():549] communicating current version
2022-02-09 06:28:47,320 INFO    MainThread:2844 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-09 06:28:47,320 INFO    MainThread:2844 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-09 06:28:47,589 INFO    MainThread:2844 [wandb_init.py:init():592] starting run threads in backend
2022-02-09 06:28:47,823 INFO    MainThread:2844 [wandb_run.py:_console_start():1816] atexit reg
2022-02-09 06:28:47,825 INFO    MainThread:2844 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-02-09 06:28:47,826 INFO    MainThread:2844 [wandb_run.py:_redirect():1695] Redirecting console.
2022-02-09 06:28:47,832 INFO    MainThread:2844 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-09 06:28:47,832 INFO    MainThread:2844 [wandb_init.py:init():619] run started, returning control to user process
2022-02-09 06:28:51,631 INFO    MainThread:2844 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-02-09 06:28:51,632 INFO    MainThread:2844 [wandb_run.py:_restore():1758] restore
2022-02-09 06:28:53,884 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-09 06:28:54,081 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-09 06:28:54,342 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-09 06:28:54,624 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 5277
}

2022-02-09 06:28:54,730 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 5303
}

2022-02-09 06:28:54,833 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 5303
}

2022-02-09 06:28:54,936 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1004
  total_bytes: 5303
}

2022-02-09 06:28:55,039 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}

2022-02-09 06:28:55,143 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}

2022-02-09 06:28:55,247 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}

2022-02-09 06:28:55,351 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}

2022-02-09 06:28:55,455 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}

2022-02-09 06:28:55,919 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}

2022-02-09 06:28:56,463 INFO    MainThread:2844 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5303
  total_bytes: 5303
}
local_info {
}

2022-02-09 06:28:57,615 INFO    MainThread:2844 [wandb_run.py:_append_files():2184] logging synced files
